modules/service/src/main/scala/lucuma/odb/graphql/mapping/GmosNorthStaticMapping.scala

```diff
@@ -19,14 +19,4 @@
        )
      )
 
-   // The GmosNorthStatic type appears in the middle of a computed JSON result.
-   // In order to avoid finding this mapping in that case, we need to be
-   // explicit about when to use this mapping.
-   lazy val GmosNorthStaticMapping: TypeMapping =
-     SwitchMapping(
-       GmosNorthStaticType,
-       List(
-         GmosNorthVisitType / "static" -> mapping
-       )
-     )
  }
```

Comment:
This looks good.  We should probably remove this hack from `GmosSouthStaticMapping` at the same time?

Reply:
Oh, yes. I didn't know there was one. I'll do that.